bin/pycbc_make_skymap

```diff
@@ -419,13 +419,14 @@ def main(trig_time, mass1, mass2, spin1z, spin2z, f_low, f_upper, sample_rate,
         bs_approximant = 'TaylorF2'
 
     # run BAYESTAR to generate the skymap
-
     cmd = ['bayestar-localize-coincs',
-           ligolw_file_path,
-           ligolw_file_path, 
-           '--waveform', str(bs_approximant),
-           '--f-low', str(f_low),
-           '-o', tmpdir]
+               ligolw_file_path,
+               ligolw_file_path, 
+               '--waveform', str(bs_approximant),
+               '--f-low', str(f_low),
+               '-o', tmpdir]
```

Comment:
@PAduverne one last request, could you please revert this whitespace change? The previous indentation style is what we usually go with.